### PR TITLE
V4.0.4

### PR DESCRIPTION
**Pull request by girichinna27**
OP-19849: Upgrade Instructions from 4.0.3 to 4.0.4

### CODE REVIEW

File: scripts/oes-data-migration-scripts/migration_v4.0.3_to_v4.0.4.yaml [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+kind: Secret
+metadata:
+  labels:
+    app: oes
+    app.kubernetes.io/managed-by: Helm
```
Comment (saitejaopsmx) [inline scripts/oes-data-migration-scripts/migration_v4.0.3_to_v4.0.4.yaml:8]: Remove app.kubernetes.io/managed-by: Helm , heritage: Helm in all files

File: scripts/oes-data-migration-scripts/migration_v4.0.3_to_v4.0.4.yaml [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+data:
+  audit-local.yml: c3ByaW5nOgogIGRhdGFzb3VyY2U6CiAgICB1cmw6IGpkYmM6cG9zdGdyZXNxbDovL29lcy1kYjo1NDMyL2F1ZGl0ZGIKICAgIHVzZXJuYW1lOiAncG9zdGdyZXMnCiAgICBwYXNzd29yZDogJ25ldHdvcmtzMTIzJwpsb2dnaW5nOgogICAgbGV2ZWw6CiAgICAgIGNvbS5vcHNteC5hdWRpdHNlcnZpY2U6IElORk8KZGI6CiAgbWlncmF0aW9uOgogICAgZW5hYmxlZDogdHJ1ZQptZXNzYWdlLWJyb2tlcjoKICBlbmFibGVkOiBmYWxzZQogIHVzZXJuYW1lOiAncmFiYml0bXEnCiAgcGFzc3dvcmQ6ICdOZXR3b3JrczEyMycKICBob3N0OiByYWJiaXRtcS1zZXJ2aWNlCiAgcG9ydDogNTY3MgogIGVuZHBvaW50OgogICAgbmFtZTogcmFiYml0bXEKc3RhbmRhcmRFcnJvckNvZGVzOgogIGZpbGVQYXRoOiAvb3BzbXgvY29uZi9zdGFuZGFyZC1lcnJvci1jb2RlLmNzdgpmZWlnbjoKICBjbGllbnQ6CiAgICBwbGF0Zm9ybXNlcnZpY2U6CiAgICAgIG5hbWU6IHBsYXRmb3Jtc2VydmljZQogICAgICB1cmw6IGh0dHA6Ly9vZXMtcGxhdGZvcm06ODA5NQogICAgYXVkaXRjbGllbnRzZXJ2aWNlOgogICAgICBuYW1lOiBhdWRpdGNsaWVudHNlcnZpY2UKICAgICAgdXJsOiBodHRwOi8vb2VzLWF1ZGl0LWNsaWVudDo4MDk4CiAgICBvZXM6CiAgICAgIHVybDogaHR0cDovL29lcy1zYXBvcjo4MDg1CiAgICBhdXRvcGlsb3Q6CiAgICAgIHVybDogaHR0cDovL29lcy1hdXRvcGlsb3Q6ODA5MAogICAgdmlzaWJpbGl0eXNlcnZpY2U6CiAgICAgIHVybDogaHR0cDovL29lcy12aXNpYmlsaXR5OjgwOTYKICAgIGRhc2hib2FyZDoKICAgICAgdXJsOiBodHRwOi8vb2VzLWRhc2hib2FyZDo4MDk0
+kind: Secret
+metadata:
+  labels:
+    app: oes
```
Comment (saitejaopsmx) [inline scripts/oes-data-migration-scripts/migration_v4.0.3_to_v4.0.4.yaml:7]: Change the app: oes to different name ... Do not give the value as oes

File: scripts/oes-data-migration-scripts/migration_v4.0.3_to_v4.0.4.yaml [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+      terminationGracePeriodSeconds: 30
+      volumes:
+      - name: audit-config-volume
+        secret:
+          defaultMode: 420
+          items:
+          - key: audit-local.yml
+            path: audit-local.yml
+          secretName: oes-audit-service-config-new
+      - name: bootstrap-config-volume
```
Comment (saitejaopsmx) [inline scripts/oes-data-migration-scripts/migration_v4.0.3_to_v4.0.4.yaml:129]: is this required ??

**Review by rahul-chekuri — APPROVED** [review]

**Comment by saitejaopsmx** [comment]
@girichinna27  Please resolve the comments

File: scripts/oes-data-migration-scripts/migration_v4.0.3_to_v4.0.4.yaml (hunk 2 above)
Comment (girichinna27) [reply scripts/oes-data-migration-scripts/migration_v4.0.3_to_v4.0.4.yaml]: changed the app value to: auditservice-new in all the 3 files

File: scripts/oes-data-migration-scripts/migration_v4.0.3_to_v4.0.4.yaml (hunk 1 above)
Comment (girichinna27) [reply scripts/oes-data-migration-scripts/migration_v4.0.3_to_v4.0.4.yaml]: Removed the above line from all the places

File: scripts/oes-data-migration-scripts/migration_v4.0.3_to_v4.0.4.yaml (hunk 3 above)
Comment (girichinna27) [reply scripts/oes-data-migration-scripts/migration_v4.0.3_to_v4.0.4.yaml:126]: Checked with developer(Rahul) - those 2 are required.

**Pull request closed without merging**